README.md

```diff
@@ -16,24 +16,6 @@ suitable reference.
 [Chalkidiki]: https://en.wikipedia.org/wiki/Chalkidiki
 [Aristotle]: https://en.wikipedia.org/wiki/Aristotle
 
-## Blog posts
-
-Here are some blog posts talking about chalk:
-
-- [Lowering Rust Traits to Logic](http://smallcultfollowing.com/babysteps/blog/2017/01/26/lowering-rust-traits-to-logic/)
-    - Explains the basic concepts at play
-- [Unification in Chalk, Part 1](http://smallcultfollowing.com/babysteps/blog/2017/03/25/unification-in-chalk-part-1/)
-    - An introduction to unification
-- [Unification in Chalk, Part 2](http://smallcultfollowing.com/babysteps/blog/2017/04/23/unification-in-chalk-part-2/)
-    - Extending the system for associated types
-- [Negative reasoning in Chalk](http://aturon.github.io/blog/2017/04/24/negative-chalk/)
-    - How to prove that something is not true
-- [Query structure in chalk](http://smallcultfollowing.com/babysteps/blog/2017/05/25/query-structure-in-chalk/)
-    - The basic chalk query structure, with pointers into the chalk implementation
-- [Cyclic queries in chalk](http://smallcultfollowing.com/babysteps/blog/2017/09/12/tabling-handling-cyclic-queries-in-chalk/)
-    - Handling cyclic relations and enabling the implementation of implied bounds and other long-desired features in an elegant way
-
-## Documentation
-
-In addition to the blog posts there is a [glossary](GLOSSARY.md) explaining some
-of the terminology used in *chalk*.
+## Contributing
+If you are interested in contributing to chalk, please take a look at the
+[Contribution guide](CONTRIBUTING.md).
```

Comment:
Hmm, I think it's kind of sad to bury this information in the CONTRIBUTING.md file, which I suspect many will not read, and likely not read in full. I'm not sure if it needs to live in the README, but at least we should link directly to the section in the contributing file where stuff can be found, and keep the link to the glossary etc.

Reply:
Agreed. I had the same thought later on.